.github/workflows/ci-docker-build.yml

```diff
@@ -32,11 +39,14 @@ jobs:
     - name: Install tools
       run: make install-ci
 
-    - name: Build docker images
-      run: make docker
```

Comment:
what happens to `make docker` target now?

Reply:
This target is unused now, I've removed it from Makefile.